packages/loader/container-utils/src/test/error.spec.ts

```diff
@@ -9,98 +9,16 @@ import { strict as assert } from "assert";
 import { ContainerErrorType } from "@fluidframework/container-definitions";
 import { isILoggingError, LoggingError, normalizeError } from "@fluidframework/telemetry-utils";
 import { ISequencedDocumentMessage } from "@fluidframework/protocol-definitions";
-import { DataCorruptionError, DataProcessingError, GenericError } from "../error";
-
-// NOTE about this (temporary) alias:
-// CreateContainerError has been removed, with most call sites now using normalizeError.
-// This represents some small behavior changes, highlighted by the diffs in these tests.
-// They should be removed in a follow-up PR since they're redudnant with normalizeError's tests
```

Comment:
@tyler-cai-microsoft - You asked why I deleted all those tests.  This comment explains it.